### DISCUSSION TITLE
Maximizing the log likelihood in Training section in Published paper

### DISCUSSION THREAD

**Discussion by sege01** [zalandoresearch/pytorch-ts#86] [General]
Hi @kashif ,

I was referring to the paper and in the training section it has the below equation where we maximize the log-likelihood
![training](https://user-images.githubusercontent.com/85530006/145412642-6a90b6cb-36ab-4e37-b4e3-1a103d1fac57.PNG)
) 
where D is the batch of time series.
Could you please help in understanding why D is used as |D| ? What does this indicate?

Thank you.

**Comment by kashif** [comment]
ah yes what i mean here is the size of the batch so $\mathcal{D}$ is the batch and $|\mathcal{D}|$ is the batch size... hope that helps? I guess its a fancy way of saying `loss.mean()` 😄

**Reply by sege01** [reply]
Thank you so much @kashif .  :)
